speech_api/app/config.py

```diff
@@ -0,0 +1,4 @@
+import os
+
+PREFERRED_MODEL = os.getenv("PREFERRED_MODEL", "base")
```

Comment:
If we decide to make whisper model version (base, small, etc., ..) a build argument not runtime argument then we can just remove this env var?

Reply:
yes then we can remove it